docs/logic-apps/authentication.md

```diff
@@ -44,6 +44,24 @@ using (var logicApp = await LogicAppClient.CreateAsync(resourceGroup, logicAppNa
 }
 ```
 
+Or, alternatively you can choose to store the **client secret** in a secret store and retrieve it with an `ISecretProvider` instance.
+
+```csharp
+string clientSecretKey = "my-client-secret-key";
```

Comment:
This is a confusing name, if I can use client secret through secret provider why are we passing it here? This is for the authentication of the service provider then or is it actually the name of the client secret his secret in the provider?

Reply:
Yes, updated according to other comments. Resolving.